### DISCUSSION TITLE
Structure of this repository

### DISCUSSION THREAD

**Discussion by KD7LRJ** [KD7LRJ/3D-LDS-Temples#5] [Ideas]
#### File types
Since we only have .STL files right now, there is a single folder for them. I expect .3MF or other types might be useful in the future. Should we create folders for each type of file or folders for each temple with the various types of files under that?

Are there any other things we should do with the folder/directory structure?

#### Other modelers
- Should this repo include contributions from other 3D modelers?
- Should it contain other temple related models (hollow for lighting, Christmas ornaments, keychains, etc.)?

Comments welcome!

**Comment by KD7LRJ** [comment]
I'm leaning towards creating a folder for each temple for several reasons:
- Temple gets renovated/rebuilt
  - Nauvoo
  - Ogden (we already have both models for this one)
  - Provo
- Different types of models
  - Christmas ornament
  - Key chain
  - Hollow to put a light inside
  - Resin vs FDM vs ??
  - Split into multiple pieces for small printers

**Reply by mealilo** [reply]
Personally, I think creating the folder for each temple like you suggested. It doesn't make sense to me to group by file type
